Automattic/newspack-plugin
# feat/fivetran

### All Submissions:






### Changes proposed in this Pull Request:



Adds an integration with [Fivetran](fivetran.com/), via a proxy.

### How to test the changes in this Pull Request:

1. Visit the Connections wizard, observe that there's only the WPCOM connection (and Google if configured)
2. Add the `NEWSPACK_FIVETRAN_PROXY` environment variable 
3. Reload the site, observe that there's a "Fivetran" section
4. Connect the services, observe the cards displaying the connector state (e.g. "connected, scheduled") after connection

### Other information:
